cpp/ql/src/Critical/GlobalUseBeforeInit.ql

```diff
@@ -98,10 +98,25 @@ predicate callReaches(Call call, ControlFlowNode successor) {
   )
 }
 
+// To avoid many false alarms like `static int a = 1;`
+predicate initialisedAtDeclaration(GlobalVariable v) {
+  exists(VariableDeclarationEntry vde |
+    vde = v.getDefinition()
+    and vde.isDefinition()
+  )
+}
```

Comment:
I think this is equivalent and slightly more clear about what it achieves.
```suggestion
predicate initialisedAtDeclaration(GlobalVariable v) { exists(v.getInitializer()) }
```

Reply:
Fixed in https://github.com/github/codeql/pull/16149/commits/132bb9f1d6cb630466cd4649404203fc80a857c2